src/Neos.NeosIo.FeatureList/Resources/Private/Fusion/Presentation/Molecule/Feature.fusion

```diff
@@ -0,0 +1,52 @@
+prototype(Neos.NeosIo.FeatureList:Molecule.Feature) < prototype(Neos.Fusion:Component) {
+
+	@styleguide {
+		props {
+			featureData = Neos.NeosIo.FeatureList:Molecule.Feature.ExampleData
+		}
+	}
+
+	featureData = ${[]}
+
+	_neosAvatarUrl = Neos.Fusion:ResourceUri {
+		path = 'resource://Neos.NeosIo.FeatureList/Public/Images/NeosAvatars/neos_avatar_primary.svg'
+	}
+
+	renderer = afx`
+		<div class="feature" data-isCoreFeature={props.featureData.isCoreFeature} data-isCommunityFeature={props.featureData.isCommunityFeature}>
+			<div class="feature__title">{props.featureData.title}</div>
+			<div class="feature__core">
+				<img src={props._neosAvatarUrl} @if.isCoreFeature={props.featureData.isCoreFeature} alt="" title="Neos Core Feature" />
```

Comment:
Shouldn't the `@if` be on the div around that?

Reply:
Not with the current styling, maybe this can be improved when more competent fe-devs update this in step 2.

Reply:
Currently the layout expects that the place is occupied wether or not the icon is there.